### PR TITLE
More accurate description of R package requirements

### PR DESCRIPTION
**Pull request by lvclark**
I have added a more accurate list of what is required for an R package to build.  Although in the episode, the man directory and NAMESPACE file are created by roxygen, they are mandatory and should not be lumped into "optional" files and directories.  (http://r-pkgs.had.co.nz/package.html doesn't indicate that the man folder is strictly required, but I'm not sure how one can have an R package without documenting its functions.  It would fail `R CMD check`, at least.)

### CODE REVIEW

**Comment by lvclark** [comment]
I also added a diagram to show what the package structure should look like when it's done.  (Sorry for multiple changes in one pull request; still learning how to do this properly.)

**Comment by katrinleinweber** [comment]
Hello, and thanks for these updates :-) I'm going to add a few in-line comments in the diff, so we can discuss and soon merge your clarifications.

**Comment by katrinleinweber** [comment]
Dear @lvclark, 

how do you feel about the review comments? Shall we merge it as is? 

Kind regards,
Katrin

**Comment by lvclark** [comment]
Where can I view the review comments?

**Comment by katrinleinweber** [comment]
Hm. GitHub shows them to me right above here. But they are definitely under [Files Changed](https://github.com/swcarpentry/r-novice-inflammation/pull/316/files).

**Comment by lvclark** [comment]
Sorry for the delay.  I was out of the country but am back now.  I still can't see the review comments.  Maybe I don't have the right privileges?  Attached are a couple screen shots of what the Comments and Files Changed pages look like to me.
lvcpullrequest1
lvcpullrequest2

File: _episodes_rmd/08-making-packages-R.Rmd [hunk 1]
```diff
@@ -12,7 +12,7 @@ objectives:
 
 keypoints:
 - "A package is the basic unit of reusability in R."
-- "Every package must have a DESCRIPTION file and an R directory containing code."
+- "Every package must have a DESCRIPTION file, a NAMESPACE file, an R directory containing code, and a man directory containing documentation."
```
[the remark this answers is not in the document]
Comment (katrinleinweber) [reply _episodes_rmd/08-making-packages-R.Rmd]: Wow! This comment was "pending". I never knew GitHub could save comments in such a draft mode, sorry!

**Comment by lvclark** [comment]
Sounds fine to me!

**Comment by katrinleinweber** [comment]
Which one sounds fine ;-) Your original or my change suggestion? I'll add a specific suggestion.

**Comment by katrinleinweber** [comment]
You're the native speaker, so it's my task to rephrase something that sounds strange ;-)

**Comment by lvclark** [comment]
Perfect!

**Comment by katrinleinweber** [comment]
🎉 Thank you!

**Pull request merged**